### DISCUSSION TITLE
folder

### DISCUSSION THREAD

**Discussion by ops244** [blakeblackshear/frigate#10877] [General]
please tell me how do I change the recording folder from (...media/frigate), for example to (...clous/frigate)?

**Comment by NickM-27** [comment]
Depends how you're running frigate. If you're using docker then you just change change the docker command volume location. 

If you're using HA addon then there is no way

**Comment by ops244** [comment]
Just the second + samba share nas addon

**Reply by NickM-27** [reply]
there is no way to change this per addon requirements

**Comment by ops244** [comment]
the fact is that the addon "samba share nas" mounts the disk into the system itself and a new folder with this disk "cloud" appears at the same address as config media share.... that's what I would like to keep in it

**Reply by NickM-27** [reply]
right, that isn't possible. Like the instructions say you can only mount on /media/frigate so the share needs to be added under that name into HA OS

**Comment by ops244** [comment]
well, a lot of things are written, there is an addon "transmission nas", it saves downloaded torrents to whatever folder you want)))
